### ISSUE TITLE
HTTP 406 error code

### ISSUE THREAD

**Issue by pramodhk** [sendgrid/sendgrid-nodejs#1249]
Hi,

I am getting this error when the API hits

error: error: Error: SyntaxError: Unexpected token < in JSON at position 0 JSONPARSEERROR when parsing: <html>
<head><title>406 Not Acceptable</title></head>
<body bgcolor="white">
<center><h1>406 Not Acceptable</h1></center>
<hr><center>nginx</center>
</body>
</html>

**Comment by JenniferMah** [comment]
Hi @pramodhk
It looks like there is an error occurring with some of the JSON. Can you use our troubleshooting guide to help [log the request body](https://github.com/sendgrid/sendgrid-nodejs/blob/main/TROUBLESHOOTING.md#viewing-the-request-body) along with the [full SendGrid error output](https://github.com/sendgrid/sendgrid-nodejs/blob/main/docs/use-cases/success-failure-errors.md) and include both outputs in your response? Can you also let us know what version of the `sendgrid-node` helper library you are using?

**Comment by sigv** [comment]
We saw requests for whitelabel settings return 406s for a bit earlier today, and I stumbled upon this ticket. However the bug appears to have resolved itself for us now. This leads me to believe something might be going wrong on the actual backend service side, potentially with regards to how headers, or accepted output formatting flags themselves, are passed around.

The following snippet produced `Error: Not Acceptable (406).` for us:

```
const client = require('@sendgrid/client')
client.setApiKey(keys('sendgrid'))
client.request({
  method: 'GET',
  url: '/v3/whitelabel/domains',
  qs: { domain: 'example.com' }
}).catch(err => {
  console.error(`Error: ${err}.`)
})
```
